app/design/frontend/Magento/luma/Magento_Checkout/web/css/source/module/checkout/_progress-bar.less

```diff
@@ -137,12 +133,12 @@
                 content: counter(i);
                 counter-increment: i;
                 .lib-typography(
```

Comment:
you should clean up this properties of lib mixin to inline instead break every param into new line